### DISCUSSION TITLE
Clear Shopping Cart not working with Breeze

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#152] [1. Help] [answered]
Hi @vovayatsyuk 

We are using a breeze theme for our site and the "Clear Shopping Cart" button is not working on the cart page, also not working with breeze blank.

we are using: 
Adobe Commerce ver. 2.4.4-p4

Thanks,
Pankaj

**Comment by pk-pankaj** [comment]
When I exclude URL  /checkout/cart/ then it works fine

**Comment by vovayatsyuk** [comment]
Thanks! Opened an issue about missing component: https://github.com/breezefront/module-breeze/issues/48

**Reply by pk-pankaj** [reply]
Thanks @vovayatsyuk 

When we can expect this to work
We have an issue on the live site, and the client chasing us.

Thanks,
Pankaj

**Reply by vovayatsyuk** [reply]
Can't promise anything. Right now I'm working on another task.

Meanwhile, if it's a critical issue for you, you can integrate it on your own.

**Reply by pk-pankaj** [reply]
Hi @vovayatsyuk 
We have integrated. Thanks for the help

Thanks,
Pankaj

**Comment by vovayatsyuk** [comment, marked as answer]
Just added the missing integration: https://github.com/breezefront/module-breeze/commit/db7ec5720238ce6b4944714d4cc158eccd726734
